span-normalizer/helm/templates/span-normalizer-config.yaml

```diff
@@ -106,3 +106,14 @@ data:
         expireAfterWriteDuration = {{ .Values.spanNormalizerConfig.excludeSpanRulesConfig.cache.expireAfterWriteDuration }}
       }
     }
+    {{- if hasKey .Values.spanNormalizerConfig "rateLimitConfig" }}
+      rate.limit.config = [
+        {{- range $k,$v := $.Values.spanNormalizerConfig.rateLimitConfig }}
```

Comment:
Did you try with sample values? rate.limit.config is a list of dict/object. What does a range with key/value mean?

Reply:
Yes Avinash. In this case, the key would be the index of the object and the value would be the object. It is being rendered as follows.
```rate.limit.config = [
          {
            tenantId = tenant1
            groupingKey = servicename
            maxSpansPerMinute = 100
          },
          {
            tenantId = tenant2
            groupingKey = http.method
            maxSpansPerMinute = 1000
          },
      ]
```